src/Voron/Impl/Journal/WriteAheadJournal.cs

```diff
@@ -1833,7 +1835,7 @@ private CompressedPagesResult PrepareToWriteToJournal(LowLevelTransaction tx, IP
             var pagesEncountered = 0;
             foreach (var txPage in txPages)
             {
-                var scratchPage = tx.Environment.ScratchBufferPool.AcquirePagePointerWithOverflowHandling(tx, txPage.ScratchFileNumber, txPage.PositionInScratchBuffer);
+                var scratchPage = tx.Environment.ScratchBufferPool.AcquirePagePointerWithOverflowHandling(tx, txPage.ScratchFileNumber, txPage.PositionInScratchBuffer, pagerState: null);
```

Comment:
Here it's okay to pass `null` because we're under write transaction lock so we can use pager state that the pager instance has. Nobody will modify / release it meanwhile.